cmds/boot/boot.go

```diff
@@ -363,6 +364,33 @@ func kexecLoad(grubConfPath string, grub []string, mountPoint string) error {
 	}
 	// defer ramfs.Close()
 
+	// if /tmp/rsdp file exist we must get the value and add it to the be.cmdline parameter
+	// this will allow the kernel to properly read the ACPI table
+	if _, err := os.Stat("/tmp/rsdp"); err == nil {
```

Comment:
this seems like too much work to me.
b, err := ioutil.ReadFile("/tmp/extracmdline")
if err != nil {
you can check for err.IsNotExist() or whatever it is
return
}

be.cmdline = append(be.cmdline, strings.Split(string(b), "\n"))